typify-impl/src/value.rs

```diff
@@ -60,7 +60,7 @@ impl TypeEntry {
                     .unwrap()
                     .output_value(type_space, value);
                 let ident = format_ident!("{}", name);
-                quote! { super::#ident ( #inner )}
+                quote! { #ident ( #inner )}
```

Comment:
related to #137, I think that the "path" needs to be a parameter to `output_value` -- simply deleting `super::` will break other cases (that apparently are untested!)